.github/workflows/ci.yml

```diff
@@ -13,11 +13,10 @@ jobs:
           - beta
           - nightly
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions-rs/toolchain@v1
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
What is the point of switching to a different action here?

Reply:
For the `actions/checkout@v3`, just keeping up.

For `actions-rs/toolcahin@v1`, it isn't maintained in a long time, and so there are a lot of warnings about deprecated functionality. Compare the warnings at the bottom on the right side of: https://github.com/Gilnaa/memoffset/actions/runs/5457597943 vs https://github.com/Gilnaa/memoffset/actions/runs/5594996285

Reply:
It is an ecosystem wide thing but seemingly unknown ...